influxql/parser.go

```diff
@@ -372,20 +374,20 @@ func (p *Parser) parseInt(min, max int) (int, error) {
 	return n, nil
 }
 
-// parseUInt32 parses a string and returns a 32-bit unsigned integer literal.
-func (p *Parser) parseUInt32() (uint32, error) {
+// parseUInt parses a string and returns a 64-bit unsigned integer.
+func (p *Parser) parseUInt(base int, bitSize int) (uint64, error) {
```

Comment:
Since you are casting to a uint32 after the call to `parseUInt`, why not just always parse everything as 64 bit, and avoid all the extra baggage of the base/bit size at all.